docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -1563,6 +1563,380 @@ properly by the operating system.
 
 1. Pressing the button once should increase the keyboard backlight.
 
+## ECR025.001 Permanent keyboard illumination after cold-boot (Firmware)
+
+**Test description**
+
+This test aims to verify that after enabling permament illumination of keyboard
+and performing cold-boot, keyboard still illuminates in firmware.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `BIOS_SETUP_KEY` = `Esc`
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../generic-test-setup/#firmware)
+
+**Test steps**
+
+1. Power on the DUT and press the `BIOS_SETUP_KEY` to enter the setup menu.
+1. Set keyboard brightness and color to arbitrary settings.
+1. Disconnect power source, and remove battery if present.
+1. Connect power and battery again.
+1. Power on the DUT again and press the `BIOS_SETUP_KEY` to enter the setup
+    menu.
+
+**Expected result**
+
+1. After cold-boot keyboard brightness and colors settings remain the same.
+
+## ECR025.002 Permanent keyboard illumination after cold-boot (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that after enabling permament illumination of keyboard
```

Comment:
Permanent illumination is a constant function, there's no option to turn it on/off.

Reply:
Updated